tests/horn/pos/scrape02.smt2

```diff
@@ -0,0 +1,109 @@
+(fixpoint "--scrape=both")
```

Comment:
These tests strike me as too large for debugging if they ever stop passing.

Could we have smaller tests that still demonstrate that qualifiers are scraped from where they should?